js/engine_number.js

```diff
@@ -39,6 +39,16 @@ class EngineNumber {
     const self = this;
     return self._units;
   }
+
+  /**
+   * Check if this number has equipment units.
+   *
+   * @returns {boolean} True if the units represent equipment units.
+   */
+  hasEquipmentUnits() {
```

Comment:
Thanks! Before finishing, please check that there are not additional places where `hasEquipmentUnits` should be used instead of `startsWith("unit")`